bot/plugins/languages.py

```diff
@@ -0,0 +1,19 @@
+import crescent
+import hikari
+
+from bot.app import Plugin
+from bot.constants import EMBED_COLOR
+
+plugin = Plugin()
+
+
+@plugin.include
+@crescent.command
+async def languages(ctx: crescent.Context) -> None:
+    lang_names = list(
+        f"`{runtime}`" for runtime in plugin.model.manager.runtimes.run if runtime
+    )
+    embed = hikari.Embed(
+        title="Supported Languages", description=",".join(lang_names), color=EMBED_COLOR
+    )
```

Comment:
```suggestion
    embed = hikari.Embed(
        title="Supported Languages", description=", ".join(lang_names), color=EMBED_COLOR
    )
```

Reply:
I don't think the space improves the readability since everthing is already in a code block.

Without
![image](https://github.com/CircuitSacul/io/assets/65521138/30af7a71-ee9b-4c04-a748-f0ff960edad9)

With
![image](https://github.com/CircuitSacul/io/assets/65521138/5b60e3a5-7010-4ae1-93ab-61fd845bdabc)